src/components/Table/TableRow.vue

```diff
@@ -13,6 +13,7 @@ export default {
     const isNestedSlot =
       defaultSlot[0].type === 'template' ||
       defaultSlot[0].type.toString() === 'Symbol(Fragment)' ||
+      defaultSlot[0].type.toString() === 'Symbol(v-fgt)' ||
```

Comment:
To be honest this is not the best way to do this but when I upgraded Vue this is the type for nested slots that I see. There is definitely a better way for us to do this but since this is currently an issue I wanted to do this to get it working faster. Im happy to hop in a call if you want and I can show this working